torch/nn/modules/activation.py

```diff
@@ -855,18 +855,21 @@ class Softshrink(Module):
         \end{cases}
 
     Args:
-        lambd: the :math:`\lambda` (must be no less than zero) value for the Softshrink formulation. Default: 0.5
+        lambd (float): The :math:`\lambda` (must be greater than or equal to zero) value for the Softshrink formulation. Default: 0.5
 
     Shape:
         - Input: :math:`(*)`, where :math:`*` means any number of dimensions.
         - Output: :math:`(*)`, same shape as the input.
 
+    Supported Dtypes:
+        - Input tensor can have float32 or float64 data type.
```

Comment:
Is this true? I see from the opinfo testing that it can support `torch.bfloat16` and `torch.float16` as well

https://github.com/pytorch/pytorch/blob/35750bf9d134726b47451d78b731017ad27fbc8b/torch/testing/_internal/common_methods_invocations.py#L14167

Reply:
> Is this true? I see from the opinfo testing that it can support `torch.bfloat16` and `torch.float16` as well
> 
> https://github.com/pytorch/pytorch/blob/35750bf9d134726b47451d78b731017ad27fbc8b/torch/testing/_internal/common_methods_invocations.py#L14167

so would be adding this will not make any sense. otherwise doing it for all documentation may take very long time